engines/petka/detection_tables.h

```diff
@@ -51,6 +51,19 @@ static const ADGameDescription gameDescriptions[] = {
 
 	},
 
+	// Red Comrades 1: Save the Galaxy (3 CD)
+	{
+		"petka1",
+		"Compressed",
+		AD_ENTRY2s("BGS1.STR",  "99832accda859e2e1daeaae0a8561aeb", 20280804,
+				   "data1.cab", "7e73a644d8b15d2fd3781de5edce0c18", 228838940),
+		Common::RU_RUS,
+		Common::kPlatformWindows,
+		ADGF_DROPPLATFORM | ADGF_DROPLANGUAGE | GF_COMPRESSED,
+		GUIO1(GUIO_NOMIDI)
+
```

Comment:
please remove this empty line

Reply:
Done, also removed empty lines for the other entries